system/lib/gl/gl.c

```diff
@@ -1689,7 +1689,6 @@ void* emscripten_GetProcAddress(const char *name_) {
   if (!strcmp(name, "glNormalPointer")) return emscripten_glNormalPointer;
```

Comment:
i have some worry about how this stuff will work with the GetProcAddress logic - it would be good to make sure all our GPA tests also run in this proxied mode.

Reply:
This PR only moves this file around for organizational purposes. The intent of obtaining function pointers to JS functions remains same - pointer to JS function is returned on the calling thread, and the called GL function then proxies if needed.